templates/applicantForm.phtml

```diff
@@ -18,50 +18,50 @@
         <form>
```

Comment:
The h1 says New applicant, need to make it dynamic so it can say edit applicant too

Reply:
same for the page title